packages/stryker-mutator-specification/src/ArrayLiteralMutatorSpec.ts

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function ArrayLiteralMutatorSpec(name: string, expectMutation: ExpectMutation) {
+
+  describe('ArrayLiteralMutator', () => {
+
+    it('should have name "ArrayLiteral"', () => {
+      expect(name).eq('ArrayLiteral');
+    });
+
+    it('should mutate filled array literals as empty arrays', () => {
+      expectMutation('[a, 1 + 1]', '[]');
+      expectMutation(`['val']`, '[]');
+    });
+
+    it('should not mutate array initializers (leave that for ArrayNewExpressionMutator)', () => {
+      expectMutation(`new Array()`);
+      expectMutation(`new Array(1, 2, 3)`);
+    });
+
+    it('should mutate empty array literals as a filled array', () => {
+      expectMutation('[]', '[[]]');
```

Comment:
Do we always want to support this mutation? Adding it to the spec project would encourage that

Reply:
Maybe we should change `['Stryker was here!']`, would be more explicit.